client/src/services/gameHelper.js

```diff
@@ -848,7 +848,14 @@ class GameHelper {
   }
 
   isDeadStar(star) {
-    return star.naturalResources != null && star.naturalResources <= 0
+    return star.naturalResources != null && star.naturalResources.economy <= 0 && star.naturalResources.industry <= 0 && star.naturalResources.science <= 0
+  }
+
+  splitResources(game) {
```

Comment:
isSplitResources?

`return game.settings.specialGalaxy.splitResources === 'enabled'`

Reply:
Changed in the file (and in StarDetail that uses it)